kolibri/plugins/oidc_provider_plugin/templates/oidc_provider/authorize_without_consent.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<body style="display:none">
+    <div class="row">
+        <div class="col-md-6 col-md-offset-3">
+        <form method="post" action="{% url 'oidc_provider:authorize' %}" id="auth_form">
+            {% csrf_token %}
+            {{ hidden_inputs }}
+        </form>
+    </div>
+</div>
+
+
+<script type="text/javascript">
+    const form = document.getElementById("auth_form")
+    const allowInput = document.createElement("input");
+    allowInput.type = "hidden";
+    allowInput.name = "allow";
+    form.appendChild(allowInput);
+    allowInput.value = "Authorize"
+    form.submit()
+
+</script>
+
+</body>
+</html>
```

Comment:
some cleanup and meta tags...

```suggestion
<!DOCTYPE html>
<html>

<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
  <title>authorizing...</title>
</head>

<body style="display: none;">

  <div>
    <div>
      <form method="post" action="{% url 'oidc_provider:authorize' %}" id="auth_form">
        {% csrf_token %}
        {{ hidden_inputs }}
      </form>
    </div>
  </div>

  <script type="text/javascript">
    const form = document.getElementById("auth_form")
    const allowInput = document.createElement("input");
    allowInput.type = "hidden";
    allowInput.name = "allow";
    form.appendChild(allowInput);
    allowInput.value = "Authorize"
    form.submit()
  </script>

</body>
</html>
```

Reply:
@indirectlylit  Thanks, good catch on the classes for the divs. I forgot to remove them.
About the meta tags, I removed them on purpose thinking it would help the page to be loaded and parsed faster, so it's not visible to the user. Do you think, taking into account the page should not be visible, they are worth?
If you think so, I don't have any problem in adding them.

Reply:
@indirectlylit  Feedback done
I have also created a client application at https://github.com/jredrejo/oidc_test_client to test the provider, and added an option to provide the client_secret when adding a new client application to the provider database.

Reply:
> About the meta tags, I removed them on purpose thinking it would help the page to be loaded and parsed faster, so it's not visible to the user. Do you think, taking into account the page should not be visible, they are worth?

I might leave out `<meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">`, but I think having the `charset` and `title` are useful

I think those specific tags would be useful